tests/Yaapii.Atoms.Tests/IO/MemoryInputTest.cs

```diff
@@ -0,0 +1,78 @@
+﻿// MIT License
+//
+// Copyright(c) 2020 ICARUS Consulting GmbH
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Xunit;
+using Yaapii.Atoms.IO;
+
+namespace Yaapii.Atoms.IO.Tests
+{
+    public sealed class MemoryInputTest
+    {
+
+        [Fact]
+        public void CreateMemoryInput()
+        {
+            var content = "This is my input!";
```

Comment:
please merge these lines. 
We want to have our classes compact. Avoid the style where you setup a variable in one line and only use it in the next. Make a block of multiple lines which are bundled together.

Reason: Every variable you introduce forces the next developer to memorize it, because it it might be called in another line below. If that is not the case, the code gets much more readable when you avoid unnecessary variables.